lib/semmle-io/.rush/temp/package-deps_build.json

```diff
@@ -0,0 +1,13 @@
+{
+  "files": {
+    "lib/semmle-io/.rush/temp/shrinkwrap-deps.json": "762eff61b1d215d6f266938fda5be80d46e5e212",
+    "lib/semmle-io/gulpfile.js/index.js": "8d874e2d20f2bf4695cbdde137a7a72a6c27737d",
+    "lib/semmle-io/package.json": "5f14bbe8dec39540adf9f9bc9fd967f652d865db",
+    "lib/semmle-io/src/digester.ts": "4d88c953647611ef5f82b66386214284d179313a",
+    "lib/semmle-io/src/index.ts": "86ce069e764ab35b0dfcf02d614b635072d144c4",
+    "lib/semmle-io/src/random-access-reader.ts": "68734b692cec47eeb157cb90250947d99edb871f",
+    "lib/semmle-io/tsconfig.json": "d2df609ec96013b8b05cec7edc86e73c3e473a55",
+    "lib/semmle-io/.rush/temp/package-deps_build.json": "610cbc3c43aaf3da66a78f33ec0844c7a6fd25a8"
+  },
+  "arguments": "gulp "
```

Comment:
is this trailing space on purpose?

Reply:
Maybe it's autogenerated and meant to be that way, not sure...

Reply:
Not sure if it's necessary. Though, since it's autogenerated, I'd rather not change it.